tests/rpc/common.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019 ChainSafe Systems (ON) Corp.
+// This file is part of gossamer.
+//
+// The gossamer library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The gossamer library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the gossamer library. If not, see <http://www.gnu.org/licenses/>.
+
+package rpc
+
+import (
+	"encoding/json"
+	"os"
+	"time"
+)
+
+var (
+	GOSSAMER_INTEGRATION_TEST_MODE = os.Getenv("GOSSAMER_INTEGRATION_TEST_MODE")
+
+	GOSSAMER_NODE_HOST = os.Getenv("GOSSAMER_NODE_HOST")
+
+	ContentTypeJSON   = "application/json"
+	dialTimeout       = 60 * time.Second
+	httpClientTimeout = 120 * time.Second
+)
+
+//TODO: json2.serverResponse should be exported and re-used instead
```

Comment:
do we have an issue open for this? I think you recommended awhile back adding issue numbers to the todo comments. might be nice to do that here but not required of course.

Reply:
oh yes, good point!